### DISCUSSION TITLE
writing/reference-use-multiple-php-versions

### DISCUSSION THREAD

**Discussion by giscus** [jonnyeom/jonnyeom.com#64] [General]
# writing/reference-use-multiple-php-versions

Reference how to use multiple php versions (php 7, php7.4 and php 8, php 8.0, php 8.1, php 8.2) on mac / macOS and Linux.

https://www.jonnyeom.com/writing/reference-use-multiple-php-versions

**Comment by Lllesliee** [comment]
I've found a tool called ServBay.dev can provide much easier way for PHP developers, especially the beginners. It includes all versions of PHP, MariaDB, and PostgreSQL, as well as Redis and Memcached. Users can run multiple versions of PHP instances all at once.
Trust me, it's made my PHP dev life a whole lot smoother. Might be worth a shot for you to check it out!
